src/DynamoCoreWpf/Views/Core/DynamoView.xaml

```diff
@@ -196,24 +196,34 @@
         <Grid.ColumnDefinitions>
             <ColumnDefinition MaxWidth="500"
                               MinWidth="1"
-                              Name="LibraryViewColumn">
-                <ColumnDefinition.Width>
+                              Name="LeftExtensionsViewColumn">
+                <!--<ColumnDefinition.Width>
                     <Binding Path="LibraryWidth"
```

Comment:
where is this property set? Is something using it and setting it externally? Now it will do nothing right?

Reply:
Preference Setting if I am correct